public/lib/tag-api-service.js

```diff
@@ -0,0 +1,73 @@
+import angular from "angular";
+import _ from 'lodash'
+
+angular
+  .module("wfTagApiService", [])
+  .service("wfTagApiService", ["$http", "$q", tagApiService]);
+
+const extractHyper = ({data})=>data
+
+function stringify(obj) {
+  //webpack won't let us use modules with es6 in them until we upgrade it
+  return _.toPairs(obj).map(([k,v])=>`${encodeURIComponent(k)}=${encodeURIComponent(v)}`).join('&')
+}
+
+function tagApiService($http, $q) {
+  const url = `${_wfConfig.tagManagerUrl}/hyper/tags`;
+
+  function tags(params) {
+    const queryString = stringify(params);
+    return $http({
+      method: "GET",
+      url: `${url}?${queryString}`,
+    }).then(extractHyper);
+  }
+
+  function getTag(id) {
+    return $http({
+      method: "GET",
+      url: `${url}/${id}`,
+    }).then(extractHyper);
+  }
+
+  function getHyperTag(hyperTag) {
```

Comment:
It'd be great to document this with an example of what it's likely to process, what it returns, and why – ATM the reader does not know what a `hyperTag` is 😁